app/components/feedback_component.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="govuk-width-container">
+  <div class="app-feedback__container">
+    <p class="govuk-body govuk-!-margin-bottom-0 govuk-!-margin-top-4">
+
+      <%= govuk_link_to(
+        'How can we improve this section? (Opens in a new tab)',
+          "https://www.apply-for-teacher-training.service.gov.uk/find-feedback?path=#{@path}&original_controller=#{@original_controller}",
```

Comment:
Probably worth moving the Apply host to Settings so that it can be configured per environment